android/detekt.yml

```diff
@@ -158,7 +158,7 @@ complexity:
   active: true
   LongMethod:
     active: true
-    threshold: 20
+    threshold: 30
```

Comment:
Double check, maybe you can simplify it.
Otherwise you will update it every time, facing problems.